community/gogs/APKBUILD

```diff
@@ -17,9 +17,14 @@ source="${pkgname}-${pkgver}.tar.gz::https://github.com/gogits/$pkgname/archive/
 	$pkgname.initd
 	$pkgname.confd
 	$pkgname.ini
-	glide.yaml.patch
 	glide.lock.patch"
-builddir="$srcdir/${pkgname}-${pkgver}"
+builddir="$srcdir/github.com/gogits/$pkgname"
+
+unpack() {
+	mkdir -p "$srcdir/github.com/gogits/"
+	tar -C "$srcdir/github.com/gogits/" -zxf $srcdir/${pkgname}-${pkgver}.tar.gz
+	mv "$srcdir/github.com/gogits/${pkgname}-${pkgver}" $builddir
+}
```

Comment:
No, because the default `unpack` doesn't unpack to a different place than `$srcdir/${pkgname}-${pkgver}` AFAIK. Am I wrong?